config/rootfs/debos/rootfs.yaml

```diff
@@ -140,13 +140,6 @@ actions:
   - action: filesystem-deploy
     description: Deploying filesystem onto image
 
-  - action: run
-    description: Move image to output directory
-    command: mv ${ARTIFACTDIR}/rootfs.ext4 ${ARTIFACTDIR}/{{ $basename -}}/
-
-  - action: run
-    command: xz -f ${ARTIFACTDIR}/{{ $basename -}}/rootfs.ext4
-
```

Comment:
As mentioned on #863, the blocks don't have to be moved as adding the `postprocess: true` attribute should ensure these steps are run after all file system changes have been made.  I think it's better if we leave them at the same place to avoid unnecessary code churn.  Also it keeps all the ext4-related steps together in the file.